### PR TITLE
Lesson 3 Assignments Slicing Lab

### PR DESCRIPTION
**Pull request by aguardia**
I changed the name of my lesson02 file, updated my series.py file and added the slicing Lab exercises. In the code bellow I don't see the changes I made to my series file.

### CODE REVIEW

**Comment by aguardia** [comment]
Added List Lab Exercise

**Comment by aguardia** [comment]
Added String Formatting Lab Exercise

File: students/AleGuardia/Lesson03/list_lab.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+# Series 3
+
+fruit_list = ["Apples", "Pears", "Oranges", "Peaches"]
+for item in fruit_list:
+    response = ""
+    while response not in ["yes", "no"]:
+        response = input("Do you like {} ? ".format(item.lower()))
+    if response == "no":
+        fruit_list.remove(item)
+print(fruit_list)
```
Comment (charlesrobison) [inline students/AleGuardia/Lesson03/list_lab.py]: Not quite working as expected - when answering 'no' the loop skips the next fruit.

File: students/AleGuardia/Lesson03/slicing_lab_exercise.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+assert remove_every_other_item(a_tuple) == (2,13,5)
+
+assert first_last_four_remove_every_other(a_string) == " sas"
+assert first_last_four_remove_every_other(a_tuple) == ()
+
+assert reverse_elements(a_string) == "gnirts a si siht"
+assert reverse_elements(a_tuple) == (32,5,12,13,54,2)
+
+assert mid_last_first(a_string) == "is a stringthis "
+assert mid_last_first(a_tuple) == (13, 12, 5, 32, 2, 54)
```
Comment (charlesrobison) [inline students/AleGuardia/Lesson03/slicing_lab_exercise.py]: Good assert statements!

**Pull request merged**